expression/aggregation/descriptor.go

```diff
@@ -172,7 +175,21 @@ func (a *AggFuncDesc) GetAggFunc() Aggregation {
 	case ast.AggFuncAvg:
 		return &avgFunction{aggFunction: aggFunc}
 	case ast.AggFuncGroupConcat:
-		return &concatFunction{aggFunction: aggFunc}
+		var s string
+		var err error
+		var maxLen uint64
+		s, err = variable.GetSessionSystemVar(ctx.GetSessionVars(), variable.GroupConcatMaxLen)
```

Comment:
Can line 181~line191 be extracted as a function?

Reply:
@XuHuaiyu  I take a look here again and found it unnecessary to call `variable.GetGlobalSystemVar` here. `variable.GetSessionSystemVar` do this already...